app/controllers/admin_controller.rb

```diff
@@ -232,6 +232,6 @@ def downgrade
   private
 
   def user_params
-    params.require(:user).permit(:display_name, :real_name, :login, :email, :account_type, :start_date, :paid_date, :user, :owner)
```

Comment:
Having two `:real_name` parameters to `permit` seems a bit off here.  I also wonder if we're actually showing a `display_name` field to admins; if so--and if it'll be obliterated by the callbacks--we should probably remove it from the admin UI.

Reply:
Yep, I should have looked closer, that is totally what is happening, and why it wasn't working for me earlier. I'll submit another commit to address that as well.